java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/MoveMembersTransformer.java

```diff
@@ -451,6 +460,9 @@ public Boolean reduce(Boolean r1, Boolean r2) {
                     return (r1 == Boolean.TRUE || r2 == Boolean.TRUE);
                 }
             };
+            if (workingCopy.getTrees().getTree(el) == null) {
+                System.err.println("!!!");
```

Comment:
Left over debug message, I guess.

Reply:
Right. Sorry, I mistakenly based this branch on a different branch, fixed. There should only be one file changed, the ClassPathProvider.